### ISSUE TITLE
Improve scrollbar contrast and visibility when highlighed by changing the color scheme

### ISSUE THREAD

**Issue by IndigoAK** [goatcorp/FFXIVQuickLauncher#411]
Currently, the highlight color of the scrollbar when viewing the patch notes for XIVLauncher is white.  The background color of the text box containing the patch notes is also white.  This creates a visibility problem for people with poor eyesight and/or contrast issues.

Instead of the current color scheme, I suggest that the default, un-highlighted color for the top and bottom arrows be changed to match the dark grey of the window and the un-highlighted color of the scrollbar indicator to remain unchanged.  However, instead of changing the highlighted/active color to white, I'm suggesting that the color be changed to the dark.

So, in summary:

Un-highlighted colors: Top/Bottom Arrows (Dark Grey), Scrollbar background (White), Scrollbar (Light Grey)
Highlight Colors: Top/Bottom Arrows (Dark Grey), Scrollbar Background (White), Scrollbar (Dark Grey)
